pxtlib/emitter/cloud.ts

```diff
@@ -182,8 +183,8 @@ namespace pxt.Cloud {
         } else {
             url = `md/${pxt.appTarget.id}/${docid.replace(/^\//, "")}?targetVersion=${encodeURIComponent(targetVersion)}`;
         }
-        if (!packaged && locale != "en") {
-            url += `&lang=${encodeURIComponent(locale)}`
+        if (locale != "en") {
+            url += `${packaged ? "?" : "&"}lang=${encodeURIComponent(locale)}`
```

Comment:
just to double-check, in the fully offline case this parameter is just ignored if the file isn't found?

Reply:
Yes, currently it is completely ignored, and with the change in pxt-electron to make use of the parameter the logic is basically 'if lang specified, try to fetch online and fall back to local copy'